addons/pos_mercury/views/pos_config_views.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="pos_config_view_form_inherit_pos_mercury" model="ir.ui.view">
+    <record id="pos_config_view_form" model="ir.ui.view">
```

Comment:
```suggestion
    <!-- TODO the inherit for
         point_of_sale.res_config_settings_view_form in this module
         uses the pos_config_view_form_inherit_pos_mercury xml id
         already. It should be renamed to
         res_config_settings_view_form_inherit_pos_mercury. -->
    <record id="pos_config_view_form_inherit_pos_mercury_2" model="ir.ui.view">
```

Reply:
Can I go ahead with your Suggestion or Do I need to Wait for Additional review from Nim?